Integrations/integration-Okta.yml

```diff
@@ -2,7 +2,6 @@ commonfields:
   id: okta
   version: -1
 name: okta
-system: true
 display: okta
```

Comment:
example of how to specify fromversion... That's what you're asking me to add, no?